docs/mvvm/source-gen/diagnostics/MVVMTK0009.md

```diff
@@ -0,0 +1,35 @@
+---
+title: MVVM Toolkit error MVVMTK0009
+author: Sergio0694
+description: MVVM Toolkit error MVVMTK0009
+keywords: community toolkit, dotnet, csharp, mvvm, net core, net standard, source generators
+dev_langs:
+  - csharp
+---
+
+# MVVM Toolkit error MVVMTK0009
+
+The `CanExecute` name in `[RelayCommand]` must refer to a valid member in its parent type.
+
+The following sample generates MVVMTK0009:
+
+```csharp
+using CommunityToolkit.Mvvm.Input;
+
+namespace MyApp;
+
+public partial class SampleViewModel
+{
+    private bool Foo => true;
+
+    // There is no member named "Bar"
+    [RelayCommand(CanExecute = "Bar")]
+    private void GreetUser()
+    {
+    }
+}
+```
+
```

Comment:
Not sure if we've tested. but how does this work with inheritance if the prop is on the base class?

```cs
public partial class BaseViewModel:
{
      private bool Bar => true;
}

public partial class SampleViewModel : BaseViewModel
{
    private bool Foo => true;
    // There is no member named "Bar"
    [RelayCommand(CanExecute = "Bar")]
    private void GreetUser()
    {
    }
}
```

Also should these classes/fields have attributes as well to make them observable?